pipeline/merger/src/main/scala/weco/pipeline/merger/rules/TargetPrecedence.scala

```diff
@@ -2,18 +2,36 @@ package weco.pipeline.merger.rules
 
 import weco.catalogue.internal_model.work.Work
 import weco.catalogue.internal_model.work.WorkState.Identified
-
-object TargetPrecedence {
-  import WorkPredicates._
-
-  // This is the canonical list of the order in which we try to select target works
-  private val targetPrecedence = Seq(
+import weco.pipeline.merger.rules.WorkPredicates.{
+  physicalSierra,
+  sierraElectronicVideo,
+  sierraWork,
+  singlePhysicalItemCalmWork,
+  teiWork,
+  WorkPredicate
+}
+object TeiTargetPrecedence extends BaseTargetPrecedence {
+  override val targetPrecedence: Seq[WorkPredicate] = Seq(
     teiWork,
     singlePhysicalItemCalmWork,
     sierraElectronicVideo,
     physicalSierra,
     sierraWork
   )
```

Comment:
```suggestion
  override val targetPrecedence: Seq[WorkPredicate] = Seq(teiWork) ++ DefaultTargetPrecedence.targetPrecedence
```

This means the TEI merger will stay in sync with the rest of the pipeline, and emphasises the difference between them.

Reply:
Tests should catch this, and the coupling makes me feel a bit weird, but makes sense in this instance.

Reply:
Is the coupling wrong? To me it feels like the TEI target precedence is a specialisation of the default target precedence.

Reply:
> and the coupling makes me feel a bit weird
Coupling which things?

Reply:
Coupling the default and tei... which I think makes sense.